File: Capitulo_2/JavaScript/C-Exercicios-Propostos(pag-50-ate-53)/exercicio-5.js
const prompt = require('prompt-sync')()

let x1 = Number(prompt(`Digite o ponto X1:`))
let y1 = Number(prompt(`Digite o ponto Y1:`))
let x2 = Number(prompt(`Digite o ponto X2:`))
let y2 = Number(prompt(`Digite o ponto Y2:`))

let d = Math.sqrt((x2-x1)**2+(y2-y1)**2)

console.log(`A ditancia entre os pontos P1 e P2:${d}`)

File: Capitulo_2/Python/C_Exercicios_Propostos(pag-50-ate-53)/exercicio-5.py
from math import sqrt

x1 = float(input('Digite o ponto X1:'))
y1 = float(input('Digite o ponto Y1:'))
x2 = float(input('Digite o ponto X2:'))
y2 = float(input('Digite o ponto Y2:'))

d = sqrt((x2-x1)**2+(y2-y1)**2)

print(f'A ditancia entre os pontos P1 e P2:{d}')